### DISCUSSION TITLE
Takım görevleri 3

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#89] [Göster ve Söyle]
[Zekaküp](https://zekakup.com) sitesinden 3 orta 2 zor soru çözeceksiniz. Takımdan herhangi birileri çözebilir. Daha önce çözülmemiş bir soru olması gerekiyor. Cevabınızın altına takım adını yazınız. Her açıklamalı ve doğru çözüm için 20 puan kazanacaksınız.

**Comment by sonsuzus** [comment]
Düzeltme: Basit soru kalmamış çözülmeyen bu yüzden 3 orta 2 zor olarak güncelledim.

**Comment by Sadik1Dev** [comment]
takım: rookies

soru:Bir bakteri kolonisi deney tabağı üzerinde yerleştirilmekte ve her gün aynı saatte gözlenmektedir. Bakteri kolonisi, her gün bir önceki gün kapladığı alanın üç katını kaplayarak büyümektedir. Koloninin yerleştirildiği 60 cm^2 lik deney tabağının yüzeyini tamamen kaplaması, 12 gün almaktadır.
İki özdeş koloninin deney tabağının yüzeyini birlikte kaplaması kaç günde tamamlanır?
```
gun=1
artis=1
kaplanan=0
katsayi=3

while gun<=12:
   artis=kaplanan*katsayi   
   
   if artis==0:
       artis=1
   kaplanan=kaplanan+artis

   print ("Gün:",gun,"artis:",artis,"kaplanan:",kaplanan)
   
   gun+=1

print ("1 koloni 12 günde kapladığı alan:",kaplanan)

kaplanacak=kaplanan

kaplanan=0
gun=1
katsayi=katsayi*2

print ("2 koloni ile kaplanacak alan:",kaplanacak,"kaplanan:",kaplanan)
while kaplanan<=kaplanacak:
   artis=kaplanan*katsayi  
   
   if artis==0:
       artis=1
   kaplanan=kaplanan+artis

   print ("Gün:",gun,"artis:",artis,"kaplanan:",kaplanan)
   gun+=1

print ("2 bakteri kolonisi gün sayısı:",gun-1)
```

**Comment by Berkek007** [comment]
Takım: Black Rhinos
Soru: Şehir şifre sorusu

konya=20
istanbul=31
muş=10
bilecik=?
bitlis=22

cevap : bilecik=32 olmalıdır. 3 sayısı kaç heceden oluştuğunu gösteriyor, 2 ise "bilecik" kelimesi içindeki "i" harfi sayısını gösteriyor. Aynı konya'da olduğu gibi=20 sayısı, 2 sayısı kaç heceden oluştuğunu, 0 ise "konya" kelimesi içindeki "i" harfi sayısını gösteriyor. Konya kelimesi "i" harfi içermediği için "0" yazılıyor.

**Comment by Berkek007** [comment]
Takım: Black Rhinos
Soru: Sayı dizisindeki soru işaretli yere

13, 19 , 52 , 99 , 117 , 184 , 208 , 295 , 396 , 432 , (?) , 595 , 736 , 891 , 945

sayı dizisindeki soru işaretli yere hangi sayı gelmelidir?

Cevap 553. Bu dizi aslında 6'nın katı olan sayılardan türemiş bir dizidir. 

Esas dizi 12,18,24,30,36,42,48,54,60,66,72,78,84,90,96 şeklindedir. 

Burada örneğin 13 sayısı (12+1).1 = 13
                             19 sayısı (18+1)1 = 19
                             52 sayısı (24+2).2 = 52 olmuştur. Yani her sayı kendi onlar basamağında bulunan rakam ile toplanıp, daha sonra bulunan sayı yine aynı onlar basamağında bulunan rakam ile çarpılarak ikinci dizi oluşmuştur. Buna göre 11.sayı 72'ye karşılık gelen sayı (72+7)+7=553 olur.

**Comment by Alperg123456** [comment]
![öldürme](https://user-images.githubusercontent.com/114610395/225412992-87395ad5-65df-4055-afe0-082b86537cdd.jpeg)


Cevap 19 ve 35

**Reply by Berkek007** [reply]
Takım: Black Rhinos

**Comment by AcelyaP** [comment]
Takım: KİttyCats
seviye: Orta
Maymun tırmanmaya başlarsa sorusu

Cevap: Maymun ipten yukarı çıktığı ve yükün karşı tarafında kaldığı sürece bir değişim olmayacaktır. Ne zaman maymun yükün olduğu tarafa geçer, o zaman yük ve maymun aşağı doğru düşecektir.

**Comment by Berkek007** [comment]
Takım: Black Rhinos
Soru: Kavanozdaki sinekler sorusu

Uçarken sineklerin ağırlıkları tartıya yansır. Kavanozun kapağı kapalı ise ve sinekler belli bir yükseklikteyse onu yukarıda tutan maddenin hava ya da cam olması sorun değildir. Her ikisi de tartıya aynı büyüklükte kuvvet uygulamaktadır.  Sinekler kavanozun içinde olduğundan ve bununla birlikte kavanozun kapağı kapalı olduğu için havanın dağılacak yeri yoktur. Sonuç olarak kavanozun tabanına çarpan hava molekülleri diğer yüzeylerine çarpanlara göre daha büyük kuvvet uyguladıkları için sineklerin kavanozun iç yüzeylerinden birine konup durdukları duruma göre daha ağır olacakmış gibi düşünülür.Aslında sineklerin kanatlarıyla tabana uyguladıkları kuvvet diğer yüzeylere uyguladıkları kuvvetlerden büyüktür ama bütün bu kuvvetlerin toplamı sineklerin ağırlığına eşittir. Dolayısıyla sinekler kavanozun içinde ister uçsun ister kavanozun dibinde olsunlar tartıdaki değer aynı kalır.

**Comment by Berkek007** [comment]
Takım: Black Rhinos
Soru: Katlanan kağıt sorusu

Bir sayfa kağıdın kalınlığı normalde 0,009906 cm yani yaklaşık 0,1 mm civarındadır. 23 kez katlarsak 1000m kalınlığa ulaşır. 30 kez katlarsak uzaya ulaşırız yani 100 km kalınlığa,51 katlama ile de  Güneş'e ulaşırız.

**Comment by akgembi12** [comment]
[Zarların 6 gelme olasılığı-Orta](https://zekakup.com/discussion/285/zarlarin-6-gelme-olasiligi#latest)
[Altı rakamlı sayılar sorusu-Orta](https://zekakup.com/discussion/289/alti-rakamli-sayilar-sorusu#latest)
Takım: On The Way
